user/common-build-problems.md

```diff
@@ -296,6 +296,10 @@ If you have a command that doesn't produce output for more than 10 minutes, you
 
 `travis_wait` writes a short line to the build log every minute for 20 minutes, extending the amount of time your command has to finish.
 
+If you need more than 20 minutes, you can prefix `travis_wait` with a custom value. For example, this would extend the time to 30 minutes:
```

Comment:
Looks good. You could clarify a little if you wanted:

If you expect the command to take than 20 minutes, prefix `travis_wait` with a greater number. For example, to extend the time to 30 minutes:

Also, we get better syntax highlighting with backtick syntax:

    ````yaml